### ISSUE TITLE
[MAS4.1.2]Name property is not properly defined for 'Accessibility' Group.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1334]
**Actual Result:**
Name property is not properly defined for "Accessibility" Group.
When focus is landing on "Accessibility" group, narrator is reading only 'group'.

**Note** - Issue repro on all controls which appearing after hit the 'Help' message.
2. URL -> Help -> Hero Card Button -> New content loaded -> Name property is not properly define and narrator reading group when focus is landing on the whole section/group. Attachment - **"MAS4.1.2_Name property for Hero card content"**
3. URL -> Help -> Video card-> New content loaded -> Name property is not properly define and narrator reading group. Attachment - **"MAS4.1.2_Name property for ‘Microsoft Band’ text"**

**Expected Result:**
Name property should be properly define for "Accessibility" Group.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Type 'Help' in the input control at the bottom of the screen and hit enter (or click on send button).
4. Open Narrator (Ctrl+Win. Key+Enter)
5. Page will upload, Navigate to the 'Accessibility' group using Tab key.
5. Observe that Name property is not properly define for 'Accessibility' Group.

**User Impact:**
If Name,State and Role property is not defined properly for controls then the users will get confuse about the purpose of that control.

**MAS Reference:**
[MAS4.1.2 - Name,Role,Value](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge
**Tool:** Narrator

**Attachment:**
1. [MAS4.1.2_Name for groups.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2571777/MAS4.1.2_Name.for.groups.zip)
2. [MAS4.1.2_Name property for Hero card content.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2584360/MAS4.1.2_Name.property.for.Hero.card.content.zip)
3. [MAS4.1.2_Name property for Microsoft Band text.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2588626/MAS4.1.2_Name.property.for.Microsoft.Band.text.zip)

**Comment by corinagum** [comment]
Try adding the title (name) to the `speak` property on the group adaptive card `MockBot index.ts ln 268` and see if this fixes the issue -- if so, this is possibly a Mock Bot issue

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG4.1.2
